### PR TITLE
Add cli args to customise templates used to render docs

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,14 +45,25 @@ And example, for flask:
...
     from acceptable import get_metadata()
     app = Flask(__name__)
     get_metadata().bind_all(app)
 
+You can now generate api metadata like so::
 
-And for django (beta) and piston:
+    acceptable metadata your.import.path > api.json
+
+This metadata can be now used to generate documentation, and provide API linting.
```
Comment (cjwatson) [inline README.rst]: s/be now/now be/ I think

File: README.rst [hunk 2]
```diff
@@ -69,21 +80,26 @@ And for django (beta) and piston:
         allowed_methods=['POST']
         ...
 
-Acceptable will generate a JSON schema representation of the form.
+Acceptable will generate a JSON schema representation of the form for documentation.
+
+To generate API metadata, you should add 'acceptable' to INSTALLED_APPS. This
+will provide with an 'acceptable' management command::
```
Comment (cjwatson) [inline README.rst]: s/with //?

File: README.rst [hunk 3]
```diff
@@ -69,21 +80,26 @@ And for django (beta) and piston:
...
 --------------------
 
 One of the goals of acceptable is to use the metadata about your api to build documentation.
 
-First, acceptable can parse your code for acceptable metadata, and generate a json version of your api metadata::
-
-    acceptable metadata path/to/files*.py > api.json
-
-Next, acceptable can transform the previously saved metadata into markdown::
+One you have your metadata in json format, as above, you can transform that into markdown documentation::
```
Comment (cjwatson) [inline README.rst]: s/One/Once/

File: acceptable/__main__.py [hunk 4]
```diff
@@ -28,11 +35,36 @@
...
+    autoescape=False,
+)
+TEMPLATES.filters['tojson'] = tojson_filter
+
+
+class ForceAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        if not namespace.update:
+            if namespace.metadata:
+                namespace.metadata.close()  # supresses resource warning
```
Comment (cjwatson) [inline acceptable/__main__.py]: "suppresses"

File: acceptable/__main__.py [hunk 5]
```diff
@@ -28,11 +35,36 @@
...
+            namespace.force = True
+
+
 def main():
     try:
         cli_args = parse_args()
         sys.exit(cli_args.func(cli_args))
     except Exception as e:
+        raise
         sys.exit(str(e))
```
Comment (cjwatson) [inline acceptable/__main__.py]: The `raise` leaves the subsequent code unreachable ...

Comment (bloodearnest) [reply acceptable/__main__.py]: gah, was debugging code, sorry will remove

**Pull request merged**